services/core-api/app/api/mines/project_summary/resources/project_summary_list.py

```diff
@@ -116,6 +116,8 @@ def post(self, mine_guid):
             project_summary.save()
             if is_minespace_user():
                 project_summary.send_project_summary_email_to_ministry(mine)
```

Comment:
This should have the same logic - only send if it goes from draft - open for both emails. 

The ministry cannot see applications that are in draft - that logic was added after the original email service was created. 

So came logic as the put: 

```
if prev_status == 'DFT' and project_summary.status_code == 'OPN':
project_summary.send_project_summary_email_to_ministry(mine)
            project_summary.send_project_summary_email_to_proponent(mine)
            ```